### ISSUE TITLE
Need sniffdog support 12 cards rig

### ISSUE THREAD

**Issue by buxstore** [Sniffdog/Sniffdogminer#201]
Need sniffdog support 12 cards rig

**Comment by Sniffdog** [comment]
please let me know which miner is not allowing you to use 12 card please

**Comment by Sniffdog** [comment]
yes sorry for the delay but enemy just updated to 1.05a for x16r and 1.04a for x16s and claims to have fixed the card issue....

you will need to unpack these zip files at the bin folder and turn them on in the miners folder because I have nevermore as default till I can get enemy to unpack properly from Mega...

x17 is back on Alexis for now until i can find an alternative to your issues with alexis....

**Comment by kodeballer** [comment]
can this be used with EthOS distro, a linux based OS? I had been using on windows 7 & 10
